docs(templates): docs for dynamic outputs and execution and task listeners (#8152)
camunda/camunda-docs: docs/components/modeler/element-templates/template-properties.md

}
```

> ## Displaying hidden entries: `entriesVisible`

Reviewer: ```suggestion
## Control visibility of default properties panel entries: `entriesVisible`
```

> By default, the element template defines the visible entries in the properties panel. All other property controls are hidden.

Reviewer: ```suggestion
By default, an applied element template causes most properties panel sections to be hidden - element template defined bindings take precedence over standard groups and entries. This behavior can be customized through the `entriesVisible` property.
```

> To bring hidden entries back, use the `entriesVisible` property.

Reviewer: ```suggestion
### Displaying all entries

To show all standard properties panel entries, set `entriesVisible=true`:
```

]
```

Instead of showing all entries, you can selectively enable specific property groups by setting `entriesVisible` to an object. Currently, the `outputs` group is supported:

```json
[
  {
    "name": "Template 1",
    "id": "sometemplate",
    "entriesVisible": {
      "outputs": true
    },
    "appliesTo": [
      "bpmn:ServiceTask"
    ],
    "properties": [
      ...
    ]
  }
]
```

> You should generally avoid using this configuration, as it removes the abstraction introduced by the template.

Reviewer: There can be good reasons why customers choose to use the default properties or hide existing ones. Let's remove this line.

```suggestion
> [!NOTE]
> As an element template author your are in control and have to ensure that default sections opened don't conflict with bindings defined by the element template.
```

]
```

### Control specific entry visibility

By passing an object to `entriesVisible` you can override the default display of certain properties panel sections.
The key of that object is the ID of a section, the value is a boolean that defines the visibility status.
The table below lists what entries may be customized and their default visibility status:

| Key       | Description            | Default visible |
| :-------- | :--------------------- | :-------------- |
| `outputs` | Output mapping section | `false`         |

```json
[
  {
    "name": "Template 1",
    "id": "sometemplate",
    "entriesVisible": {
      "outputs": true
    },
    "appliesTo": [
      "bpmn:ServiceTask"
    ],
    "properties": [
      ...
    ]
  }
]
```

> > [!NOTE]

Reviewer: @AlekseyManetov Let's check back with documentation if this is the way we add informative boxes - it is GitHub style, in our documentation it could be different.

Author: While I found a few usages of `[!NOTE]`, they aren't being formatted correctly. We should use our own [admonitions](https://github.com/camunda/camunda-docs/blob/main/howtos/technical-writing-styleguide.md#formatting-organization-and-structure-for-implementation-steps), such as `:::warning`, so I have replaced with its. Thanks for pointing that out!
